src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -267,6 +268,21 @@
             TokenTypes.BXOR_ASSIGN,
         }).collect(Collectors.toSet()));
 
+    /** List of tokens from {@link #getClassFrameWhereViolationIsFound(DetailAST)} method.*/
+    private static final List<Integer> EXPR_TOKENS = Arrays.asList(
+            TokenTypes.PLUS,
+            TokenTypes.MINUS,
+            TokenTypes.EQUAL,
+            TokenTypes.NOT_EQUAL,
+            TokenTypes.STAR,
+            TokenTypes.DIV,
+            TokenTypes.MOD,
+            TokenTypes.LT,
+            TokenTypes.GT,
+            TokenTypes.LE,
+            TokenTypes.GE
```

Comment:
Can we sort/group these somehow? Either Alphabetical or by conditions / math operators?

Reply:
I changed order to alphabetical